controllers/starter.go

```diff
@@ -89,5 +89,22 @@ func SetupAllReconcilers(mgr controllerruntime.Manager, cfg *config.OperatorConf
 		return err
 	}
 
+	// Setup tokenUpload controller if configured
+	// Important: need NotifyingTokenStorage to reconcile related SPIAccessToken
+	notifyingStorage := tokenstorage.NotifyingTokenStorage{
```

Comment:
can we move it behind `cfg.EnableTokenUpload`

Reply:
sure